src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -193,13 +193,23 @@ export async function buildMapView(id: string, dotNetReference: any, long: numbe
             if (mapObject.arcGISDefaultBasemap !== undefined &&
                 mapObject.arcGISDefaultBasemap !== null) {
                 basemap = mapObject.arcGISDefaultBasemap;
-            } else if (mapObject.basemap?.portalItem?.id !== undefined &&
-                mapObject.basemap?.portalItem?.id !== null) {
-                basemap = new Basemap({
-                    portalItem: {
-                        id: mapObject.basemap.portalItem.id
-                    }
-                });
+            } else if (hasValue(mapObject.basemap?.portalItem?.id)) {
```

Comment:
Do we need to create several different items in our portal to test these?

Reply:
not a bad idea, if we can add more to our enterprise portal.